### ISSUE TITLE
Feeder Lane (asl_fdr)

### ISSUE THREAD

**Issue by RobJN** [cyclestreets/tflcid-conversion#1]
There are two types here and you are showing just one of these in the image example. The type you show is when there is a cycle lane joining an advanced stop lane area (“reservior”). The other type is when there is no clycle lane in the approach to the ASL. Here TRUE is simply indicating the presence of those diagonal dashed lines at the corner of the ASL. Here is an example image at Oxford St/Charing Cross intersection
https://api.cyclestreets.net/v2/infrastructure.image?key=c047ed46f7b50b18&dataset=tflcid&id=RWG065687&variant=2 

## Current 

Convert geometry: | way->node, retain way
-- | --
OSM tag(s): | node: cycleway=asl, TRUE: way: cycleway=lane
OSM comment: | OSM treats ASLs as a node, but the CID has them as a  line. No tag for feeder lane exists yet, other than putting the lane on  the adjacent highway
Action note: | Add node for ASL itself, and then when TRUE add cycleway=lane to way

## Proposal

Convert geometry: | way->node
-- | --
OSM tag(s): | node: cycleway=asl, TRUE: node: asl:feeder=yes

You could even have some more detail (requires looking at the image):

- asl:feeder=yes
- asl:feeder=lane
- asl:feeder=~~stub~~gate

Edit: Updated to reflect that the short dashed lines in the image are called gates.

**Comment by RobJN** [comment]
Oh, potential data issue. I have just reviewed [TfL's guide](https://cycling.data.tfl.gov.uk/CyclingInfrastructure/documentation/asset_information_guide.pdf) and they state:

> Short dashed lines called ‘gates’ are not a feeder

However it didn't take me long to find an example. Maybe I just fluked upon a rare data entry error, but if not then we need to consider how to use this data. Will we need to review each photo...?
